packages/neo-one-server-plugin-project/src/initCommand.ts

```diff
@@ -7,37 +7,54 @@ import jsonFile from 'json-file-plus';
 import * as path from 'path';
 import { loadProjectConfig } from './utils';
 
+interface InitDataFileOpts {
+  readonly folder: string;
+  readonly name: string;
+  // tslint:disable-next-line:no-any
+  readonly data: any;
+  readonly cli: InteractiveCLIArgs['cli'];
+}
+export const initDataFile = async ({ folder, name, data, cli }: InitDataFileOpts) => {
+  const filePath = path.resolve(folder, name);
+  await fs.ensureDir(folder);
+  await fs.writeFile(
+    filePath,
+    JSON.stringify(
+      data,
+      // tslint:disable-next-line:no-null-keyword
+      null,
+      2,
+    ),
+  );
+  cli.print(`Created ${name} at ${filePath}.`);
+};
+
 export const initCommand = ({ cli }: InteractiveCLIArgs) =>
   cli.vorpal.command('init', 'Initialize neo-one in the current directory.').action(async () => {
     const cwd = process.cwd();
     const projectConfig = await loadProjectConfig(cwd);
     const tsconfigPath = path.resolve(projectConfig.paths.contracts, 'tsconfig.json');
     const contractsPath = path.dirname(tsconfigPath);
-
-    await fs.ensureDir(contractsPath);
-    await fs.writeFile(
-      tsconfigPath,
-      JSON.stringify(
-        {
-          compilerOptions: {
-            ...COMPILER_OPTIONS,
-            target: 'esnext',
-            module: 'esnext',
-            moduleResolution: 'node',
-            jsx: 'react',
-            plugins: [
-              {
-                name: '@neo-one/smart-contract-typescript-plugin',
-              },
-            ],
+    const tsData = {
+      compilerOptions: {
+        ...COMPILER_OPTIONS,
+        target: 'esnext',
+        module: 'esnext',
+        moduleResolution: 'node',
+        jsx: 'react',
+        plugins: [
+          {
+            name: '@neo-one/smart-contract-typescript-plugin',
           },
-        },
-        // tslint:disable-next-line:no-null-keyword
-        null,
-        2,
-      ),
-    );
-    cli.print(`Created tsconfig.json at ${tsconfigPath}.`);
+        ],
+      },
+    };
+    const rcData = { codegen: projectConfig.codegen };
```

Comment:
I think we actually want the .neorc file to contain the whole projectConfig, not just the codegen piece.

Reply:
I guess it's not necessary, but imo might as well be explicit since we have so few options.  Easier for them to figure out what config options they may want to change without digging through documentation if we just print the whole thing from the start.

Reply:
If this is Ideal, I can see where to get the default paths, otherwise "default paths" are not part of the schema definition.

Reply:
The default paths will already been included in the `projectConfig`.  The `projectConfig` is loaded by `loadProjectConfig` which will set default paths if they are not defined.  So, if you want to make this chance, all you need to do is pass the whole `projectConfig` instead of extracting out the codegen piece.